File: P2_B_Constant.java
import java.io.*;
import java.util.*;
class pconst
{
public static void pcon()
{
	String str;
	int flag=1;
	System.out.print("Enter the string :: ");
	Scanner s=new Scanner(System.in);
	str=s.nextLine();
	for (int i = 0; i < str.length(); i++) 
    	{
		if (!(str.charAt(i)>='0' && str.charAt(i)<='9'))
        	{
		flag=0;
		}
	}
	if(flag==1)
	System.out.println("Constant");
	else
	System.out.println("Not a Constant");
}
public static void main(String[] args)
{	
System.out.println("Saail Chavan KFPMSCCS016\n");
pcon();
}}

File: Py/P2_B_Constant.py
#Design a program to check given input is identifier, constants, reserved keywords, and operators.
#12345,  124asf
#Constants

def pcon():
    flag = True
    str_input = input("Enter the string: ")

    # Check if all characters are digits
    for char in str_input:
        if not ('0' <= char <= '9'):
            flag = False
            break

    # Print the result based on the flag
    if flag:
        print("Constant")
    else:
        print("Not a Constant")


if __name__ == "__main__":
    print("Saail Chavan KFPMSCCS016")
    pcon()
